packages/platform-sdk-profiles/src/deser.ts

```diff
@@ -0,0 +1,9 @@
+import { StorageSerializer, StorageDeserializer } from "./interfaces";
+
+export const serializer: StorageSerializer = (val: any) => {
+	return JSON.stringify(val);
```

Comment:
Will we ever have to store and serialise buffer-ish data? If so we would maybe have to use https://github.com/dominictarr/json-buffer to avoid that data being malformed.